### ISSUE TITLE
srloop produces output despite of verbose = 0 (almost mute)

### ISSUE THREAD

**Issue by rafajot** [secdev/scapy#750]
# cat testsrloop.py
```
from scapy.all import srloop,IP,ICMP

print "aa"
srloop(IP(dst="8.8.8.8") / ICMP(), verbose=0, count=1)
print "bb"

```
# python testsrloop.py
aa
bb**nd...**

_Note extra output here_

# pip freeze |grep scapy
scapy==2.3.3
#

**Comment by p-l-** [comment]
Hi,

Thanks for your report. This is bug, and it should be fixed in #757. You can apply this patch to test it.

**Comment by yang-dev** [comment]
```
    srloop(pkts, *args, **kargs)
        Send a packet at layer 3 in loop and print the answer each time
        srloop(pkts, [prn], [inter], [count], ...) --> None
```

The man page for srloop is so vague
